eclipsePlugin/build.gradle

```diff
@@ -74,6 +74,7 @@ task copyLibsForEclipse (type: Copy) {
     into "lib"
     include "*.jar"
     exclude "*xml-apis*.jar"
+    exclude "jdepend*.jar"
```

Comment:
Sorry, I missed this notification. This exclude should not be necessary. `jdepend` is a `testCompile` dependency under `:spotbugs`, and therefore not part of the default configuration being used here as dependency.

As to moving from `embeddedLibs` to compile, we would need to change the eclipse jars from `compile` to `compileOnly`, and make sure those are not transitively included in the jar, but I believe it should work just fine.